api/src/main/java/com/codeforcommunity/exceptions/TokenInvalidException.java

```diff
@@ -3,6 +3,10 @@
 import com.codeforcommunity.rest.FailureHandler;
 import io.vertx.ext.web.RoutingContext;
 
+/**
+ * Used when either an access or refresh JWT token is invalid for any reason, including when it is
+ * expired.
+ */
 public class TokenInvalidException extends HandledException {
```

Comment:
Do we not throw a TokenExpiredException anywhere? Or does invalid cover all cases?

Reply:
Invalid covers all cases